### PR TITLE
Fix e2e test after registry changes

### PR DESCRIPTION
**Pull request by feloy**
**What type of PR is this:**

/kind tests

### CODE REVIEW

File: tests/e2escenarios/e2e_test.go [hunk 1]
```diff
@@ -93,7 +93,7 @@ var _ = Describe("E2E Test", func() {
 
 			// "running odo list"
 			stdout := helper.Cmd("odo", "list", "component").ShouldPass().Out()
-			helper.MatchAllInOutput(stdout, []string{componentName, "nodejs", "Dev"})
+			helper.MatchAllInOutput(stdout, []string{componentName, "Node.js", "Dev"})
```
Comment (valaparthvi) [inline tests/e2escenarios/e2e_test.go:96]: It looks like this should be left as is. Same for the change on L119.
```
  Expected
      <string>:  â€¢  Listing components from namespace 'e2e-test47grk'  ...
      
 âœ“  Listing components from namespace 'e2e-test47grk' [455ms]
       NAME      PROJECT TYPE  RUNNING IN  MANAGED      
       * nlbvnt  nodejs        Deploy      odo (v3.1.0) 
      
  to contain substring
      <string>: Node.js
  In [It] at: /go/odo_1/tests/helper/helper_generic.go:76
```

File: tests/e2escenarios/e2e_test.go [hunk 2]
```diff
@@ -169,7 +169,7 @@ var _ = Describe("E2E Test", func() {
 			_, err := helper.RunInteractive(command, nil, func(ctx helper.InteractiveContext) {
 
 				// helper.ExpectString(ctx, "Based on the files in the current directory odo detected")
-				helper.ExpectString(ctx, "Language: javascript")
+				helper.ExpectString(ctx, "Language: javaScript")
```
Comment (valaparthvi) [inline tests/e2escenarios/e2e_test.go:172]:
```suggestion
				helper.ExpectString(ctx, "Language: JavaScript")
```

File: tests/e2escenarios/e2e_test.go (hunk 1 above)
Comment (feloy) [reply tests/e2escenarios/e2e_test.go:96]: Thanks. It shows there is an inconsistency for the PROJECT value between Dev and Deploy'ed components. I'll create an issue to follow this.

**Comment by valaparthvi** [comment]
I wonder if we should define these projectType and componentName values as constants. We have been going back and forth a lot lately, as a temporary solution, it might help to define constants.

**Comment by feloy** [comment]
/override windows-integration-test/Windows-test
PVC problem

/NoCluster-Integration-tests/NoCluster-Integration-tests
Non applicable

**Comment by feloy** [comment]
/override windows-integration-test/Windows-test
PVC problem

/override NoCluster-Integration-tests/NoCluster-Integration-tests
Non applicable

**Review by rm3l — APPROVED** [review]
/approve

**Pull request merged**